storage/api/csk_operations.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Command-line sample app demonstrating customer-supplied encryption keys.
+
+This sample demonstrates uploading an object while supplying an encryption key,
+retrieving that object's contents, and finally rotating that key to a new value.
+
+This sample is used on this page:
+
+    https://cloud.google.com/storage/docs/json_api/v1/json-api-python-samples
+
+For more information, see the README.md under /storage.
+"""
+
+import argparse
+import filecmp
+import tempfile
+
+from googleapiclient import discovery
+from googleapiclient import http
+
+from oauth2client.client import GoogleCredentials
+
+
+# You can (and should) generate your own encryption key. Here's a good way to
+# accomplish this with Python:
+#   python -c \
+#     'import base64; import os; print(base64.encodestring(os.urandom(32)))'
+# Although these keys are provided here for simplicity, please remember that it
+# is a bad idea to store your encryption keys in your source code.
+ENCRYPTION_KEY = "4RzDI0TeWa9M/nAvYH05qbCskPaSU/CFV5HeCxk0IUA="
```

Comment:
please use single quotes throughout the file, except for docstrings (continue to use `"""`).